msbuild/Xamarin.iOS.Tasks.Core/Xamarin.iOS.Tasks.Core.csproj

```diff
@@ -24,6 +25,10 @@
       <!-- We need the net472 impl, otherwise the Build agent needs to be a net5.0 app -->
       <HintPath>$(PkgMicrosoft_NET_ILLink_Tasks)\tools\net472\ILLink.Tasks.dll</HintPath>
     </Reference>
+    <Reference Include="ILStrip">
+      <!-- We need the net472 impl, otherwise the Build agent needs to be a net5.0 app -->
+      <HintPath>$(PkgMicrosoft_NET_Runtime_MonoTargets_Sdk)\tasks\net472\ILStrip.dll</HintPath>
```

Comment:
FYI, for net7 I'm looking at consolidating the MonoTargets tasks into a single assembly https://github.com/dotnet/runtime/pull/59720